src/tests/routes/index.spec.js

```diff
@@ -22,7 +23,8 @@ describe('GET /', () => {
   it('should show a welcome message for the home route', (done) => {
     chai.request(app)
       .get(`${endPoint}`)
-      .end((_err, res) => {
+      .end((err, res) => {
```

Comment:
if you're removing the _ from 'err', eslint would flag that, so you would also have to go and add a rule exception in the eslintrc file, or just stick with adding the underscore for unused variables.